src/model/RestAction.ts

```diff
@@ -218,16 +255,28 @@ class RestAction implements Action {
                                     `Header validation (${validation}): ${validationResult}`,
                                 );
                                 reject(head);
```

Comment:
Obsolete reject() call. Error creation afterwards is intended.

Reply:
removed it